updater.go

```diff
@@ -203,3 +174,41 @@ func applyUpdate(r io.Reader, publicKey ed25519.PublicKey, signature [64]byte) (
 	}
 	return opts.TargetPath, nil
 }
+
+func triggerSchedule(updater *Updater) {
+	for {
+		var next time.Duration
```

Comment:
Sorry to be picky, but `next` is a time not a duration, perhaps "offset" or "delta" or "delay" etc might work better?